src/java/org/apache/cassandra/index/sai/utils/TreeFormatter.java

```diff
@@ -48,6 +49,21 @@ public TreeFormatter(Function<T, String> toString, Function<T, Iterable<? extend
     {
         this.children = children;
         this.toString = toString;
+        this.indent = "";
+    }
+
+    /**
+     * Constructs a formatter that knows how to format trees of given type.
+     *
+     * @param toString a function that returns the text describing each tree node
+     * @param children a function that returns a list of children nodes
+     * @param indent a string used for indentation
+     */
+    public TreeFormatter(Function<T, String> toString, Function<T, Iterable<? extends T>> children, String indent)
```

Comment:
I keep on asking myself... why in the other logs we do not want it with indentation?